src/helpers.ts

```diff
@@ -233,9 +233,16 @@ export function rewriteErrorStack(stack: string, indexes: [number, number]) {
   return stack;
 }
 
-export function formatError(error: Error) {
+// formatError prefers receiving proper Errors, but since at runtime
+// non Error exceptions can be thrown, it tolerates though. The
+// redundant type Error | any expresses that.
+export function formatError(error: Error | any) {
   if (!(error instanceof Error)) {
```

Comment:
either we need to handle undefined objects here or https://github.com/elastic/synthetics/blob/41e6c2acc73baadb3582d46388ffbd5a835b4f0f/src/reporters/json.ts#L558

Otherwise even undefined objects get these errors reported which is incorrect.

Reply:
I think the clause above handles that right, or am I missing something?